main.go

```diff
@@ -0,0 +1,304 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"strconv"
+	"strings"
+
+	"github.com/Shopify/sarama"
+	"github.com/companieshouse/chs.go/avro"
+	"github.com/companieshouse/chs.go/avro/schema"
+	"github.com/companieshouse/chs.go/kafka/producer"
+	"github.com/companieshouse/kafka-management-tool/schemas"
+)
+
+// MandatoryFlags lists all flags and their types
+type MandatoryFlags struct {
+	Broker         string
+	Topic          string
+	Schema         string
+	SchemaRegistry string
+	Partition      int64
+	Offset         string
+	JsonOut        int64
```

Comment:
Change capitalisation to `JSONOut`

Reply:
done [19892e6](https://github.com/companieshouse/kafka-management-tool/pull/1/commits/19892e64d016e5bf48a8eb8634703e199485f718)